developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,27 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+=== Running tests in Foreman
+Make sure to run tests from the Foreman directory. You can pass an argument to specify the path to a directory or to a file.
+
+In order to run rubocop test, use the following command:
+[source, ruby]
+....
+bundle exec rubocop [<path>]
+....
+
+To run Foreman's tests:
+[source, ruby]
+....
+bundle exec rake test [<path>]
```

Comment:
Does this work? Isn't path interpreted as a rake task?

Reply:
You are right! Thank you for noticing!